opencti-platform/opencti-graphql/src/http/httpAuthenticatedContext.js

```diff
@@ -60,6 +61,8 @@ export const createAuthenticatedContext = async (req, res, contextName) => {
         executeContext.user_otp_validated = req.session?.user.otp_validated ?? false;
       }
       executeContext.user_inside_platform_organization = isUserInPlatformOrganization(user, settings);
+      const licenseInfo = getEnterpriseEditionInfo(settings);
```

Comment:
@richard-julien 
how it's deployed, it's visible on pyroscope, and should be changed to use a cache